e2e/tests/functional/plugins/notebook/notebookSnapshots.e2e.spec.js

```diff
@@ -161,3 +163,56 @@ test.describe('Snapshot Container tests', () => {
     }
   );
 });
+
+test.describe('Snapshot image tests', () => {
+  test.beforeEach(async ({ page }) => {
+    //Navigate to baseURL
+    await page.goto('./', { waitUntil: 'domcontentloaded' });
+
+    // Create Notebook
+    await createDomainObjectWithDefaults(page, {
+      type: NOTEBOOK_NAME
+    });
+  });
+
+  test('Can drop an image onto a notebook and create a new entry', async ({ page }) => {
+    const imageData = await fs.readFile('src/images/favicons/favicon-96x96.png');
+    const imageArray = new Uint8Array(imageData);
+    const fileData = Array.from(imageArray);
+
+    const dropTransfer = await page.evaluateHandle((data) => {
+      const dataTransfer = new DataTransfer();
+      const file = new File([new Uint8Array(data)], 'favicon-96x96.png', { type: 'image/png' });
+      dataTransfer.items.add(file);
+      return dataTransfer;
+    }, fileData);
+
+    await page.dispatchEvent('.c-notebook__drag-area', 'drop', { dataTransfer: dropTransfer });
+
+    // be sure that entry was created
+    await expect(page.getByText('favicon-96x96.png')).toBeVisible();
+
+    // click on image (need to click twice to focus)
+    await page.getByRole('img', { name: 'favicon-96x96.png thumbnail' }).click();
+    await page.getByRole('img', { name: 'favicon-96x96.png thumbnail' }).click();
+
+    // expect large image to be displayed
+    await expect(page.getByRole('dialog').getByText('favicon-96x96.png')).toBeVisible();
+
+    await page.getByLabel('Close').click();
+
+    // drop another image onto the entry
+    await page.dispatchEvent('.c-snapshots', 'drop', { dataTransfer: dropTransfer });
+
+    // expect two embedded images now
+    await expect(page.getByText('favicon-96x96.png').nth(1)).toBeVisible();
```

Comment:
should we instead check the count?

`await expect(page.getByText('favicon-96x96.png').count()).toBe(2)` ?

Reply:
much better - agreed!